server/src/models/Resources.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+
+const resource = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  link: {
+    type: String,
+    required: false,
```

Comment:
Shouldn't the link to the resource be required? A resource with only a title and description won't do much good

Reply:
Books? But I think even in that case, a link can be given. I will make this change